### PR TITLE
kernel: bump 6.1 to 6.1.92

### PR DESCRIPTION
**Pull request by zxlhhyccc**
https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.92

Removed upstreamed:
 bcm27xx/patches-6.1/950-0270-net-bcmgenet-Reset-RBUF-on-first-open.patch
 generic/backport-6.1/600-v6.9-03-net-gro-add-flush-check-in-udp_gro_receive_segment.patch
 generic/pending-6.1/681-net-bridge-fix-multicast-to-unicast-with-fraglist-GS.patch
 generic/pending-6.1/682-net-core-reject-skb_copy-_expand-for-fraglist-GSO-sk.patch
 generic/pending-6.1/684-net-bridge-fix-corrupted-ethernet-header-on-multicas.patch
 generic/pending-6.1/778-net-l2tp-drop-flow-hash-on-forward.patch
 imx/patches-6.1/002-6.2-phy-freescale-imx8m-pcie-Refine-i.MX8MM-PCIe-PHY-dri.patch

All other patches automatically rebased.

### CODE REVIEW

**Comment by hauke** [comment]
Please do not close and create a new PR, just update the existing PR #15556.

**Comment by map-b** [comment]
Actually, there are four PRs like this, the first is #15554.

**Comment by zxlhhyccc** [comment]
> Actually, there are four PRs like this, the first is #15554.

Sorry, this is the first time I've mentioned PR.

File: target/linux/x86/config-6.1 [hunk 1]
```diff
@@ -60,6 +60,7 @@ CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
 CONFIG_CPU_FREQ_STAT=y
 CONFIG_CPU_IDLE=y
 CONFIG_CPU_IDLE_GOV_LADDER=y
+CONFIG_CPU_MITIGATIONS=y
```
Comment (hauke) [inline target/linux/x86/config-6.1:63]: You can remove `SPECULATION_MITIGATIONS` from the configuration now.

Comment (zxlhhyccc) [reply target/linux/x86/config-6.1:63]: Yes，`CONFIG_SPECULATION_MITIGATIONS` It should be deleted.

**Pull request merged**